migrations/sql-migrations/dynamoToSql/run.js

```diff
@@ -65,23 +65,32 @@ const migrateProject = async (project, helper) => {
   }
 
   // Migrate all samples
-  await Promise.all(
-    samples.map(async (sample) => {
-      await helper.sqlInsertSample(experimentId, sample);
+  const insertSamplePromise = samples.map(async (sample) => {
+    const hasSample = await helper.sqlInsertSample(experimentId, sample);
 
-      const files = Object.entries(_.omit(sample.files, ['lastModified']));
+    if (!hasSample) return;
 
-      await Promise.all(
-        files.map(async ([fileName, file]) => {
-          const sampleFileUuid = uuidv4();
+    const files = Object.entries(_.omit(sample.files, ['lastModified']));
 
-          await helper.sqlInsertSampleFile(sampleFileUuid, projectUuid, sample, fileName, file);
+    await Promise.all(
+      files.map(async ([fileName, file]) => {
+        const sampleFileUuid = uuidv4();
 
-          await helper.sqlInsertSampleToSampleFileMap(sampleFileUuid, sample);
-        })
-      );
-    })
-  );
+        await helper.sqlInsertSampleFile(sampleFileUuid, projectUuid, sample, fileName, file);
+
+        await helper.sqlInsertSampleToSampleFileMap(sampleFileUuid, sample);
+      })
+    );
+  }).filter(p => p);
+
+  if(!insertSamplePromise.length) return;
```

Comment:
What is this return for? Also if it ends prematurely could we have a log warning of that maybe?